### ISSUE TITLE
Error received after loading Annotation Plugin Firefox

### ISSUE THREAD

**Issue by uskudarli** [bogaziciswe/b.w.a.t#109]
When I clicked on the icon, I got the instruction to login or register. When I registered with user "uskudarli"

I got the error;

ErrorCode -> BWAT004: chrome.storage.sync is undefined

Same happened with both Firefox and Chrome

**Comment by uskudarli** [comment]
Seems this may be duplicate user?

I tried with another user and it logged me in. Alas, I can not seem to annotate anything. All my attempts to select text and images, have not succeeded in bring up the window to specify body and motivation

**Comment by ugurhicyilmam** [comment]
I don't think it is a duplicate user issue, since server checks uniqueness of provided email addresses on registration. Also, I want to note that it should be a valid email address you are registering with.

Are you trying on Firefox? If so, can you please try it also with Chrome to help us to eliminate the possibility of a browser compatibility issue. 

Also, can you please tell us which website are you trying to annotate to help us to replicate the issue?

**Comment by uskudarli** [comment]
@ugurhicyilmam I am trying to register so I can do something.

I tried on both browsers and got the same problem. Most recenlty I tried again with Firefox and got the same error after I re-cloned your repo to make sure there is no problem with my repo. The is up to date I presume.

I am using a valid account.

**Comment by uskudarli** [comment]
I tried again with chrome. And it is giving same error. It also asks me to loging to amazon server
![registration-error](https://cloud.githubusercontent.com/assets/1347035/21741993/1e7beda4-d4ee-11e6-8ce4-60ec0aa5f5a2.png)

**Comment by ugurhicyilmam** [comment]
@uskudarli According to the logs of the authentication server, "uskudarli@gmail.com" address has already be taken. Can you please try registering with a different email address?

**Comment by ugurhicyilmam** [comment]
@uskudarli The popup that requires login to amazon is asking for your credentials to load annotations. You can provide your own credentials, or when you successfully login using extension UI, it will not appear again. It appears because we used basic authentication to handle authentication.

**Comment by uskudarli** [comment]
What credentials? I don't have any.  Since I can't register, I can't get
rid of it. Chicken and egg problem.

Also, when the extension is on, it insists on credentials all the time,
which would not be a good idea.

Can you recover my uskudarli@gmail.com related account? Or remove it?  Is
it possible?

**Comment by ugurhicyilmam** [comment]
@uskudarli I removed uskudarli@gmail.com related account. You can now use it to register a new account.

Also, I created a new issue about consistent credential asking: #110
